test/functional/helpers/constants/general.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  ecidRegex: /^[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]$/
```

Comment:
You should be able to use `^[0-9]{38}$`

Reply:
I should've mentioned you can do this too: `^\d{38}$` but it doesn't really matter.